lib/oc-tests/test_shareDir.py

```diff
@@ -95,6 +95,13 @@ def setup(step):
     check_users(config.oc_number_test_users)
 
     reset_rundir()
+    reset_server_log_file()
```

Comment:
Hi, please have a look at the preview of the hooks: https://github.com/cernbox/smashbox/tree/pre-post-hooks

Let me know if could work for you. If yes, then next we will tackle other things we discussed in Berlin.

Please do not take this as the final implementation yet.